modules/rostests/apitests/shlwapi/SHPropertyBag.cpp

```diff
@@ -168,6 +177,12 @@ static void SHPropertyBag_ReadTest(void)
     ok_long(hr, E_FAIL);
     ok_int(s_cRead, 2);
     ok_int(s_cWrite, 0);
+
+    ResetTest(VT_EMPTY, L"GUID1");
+    hr = SHPropertyBag_ReadGUID(&dummy, L"GUID1", &guid);
```

Comment:
Is this the test that checks the `/* This is by design */` thing from above? If so it would be interesting to initialize the `guid` buffer to something different from zero, then check after the function is called, whether it still contains the original data, or whether it has been zeroed out ....

Reply:
Added tests to the `SHPropertyBag` testcase.